apps/dotcms-ui/src/app/api/services/dot-configuration/dot-configuration.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from '@dotcms/dotcms-js';
+import { map, pluck, take } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotConfigurationService {
```

Comment:
We already have a [DotcmsConfigService](https://github.com/dotCMS/core-web/blob/master/libs/dotcms-js/src/lib/core/dotcms-config.service.ts#L58) and this name is too similar, we need to find another name. `DotPropertiesService` maybe? ask @erickgonzalez or @jdotcms to see if we can match the backend name of that feature.